src/DIRAC/WorkloadManagementSystem/Agent/SiteDirector.py

```diff
@@ -192,6 +193,9 @@ def beginExecution(self):
         self.sendSubmissionAccounting = self.am_getOption(
             "SendPilotSubmissionAccounting", self.sendSubmissionAccounting
         )
+        self.sendSubmissionMonitoring = self.am_getOption(
+            "SendPiolotSubmissionMonitoring", self.sendSubmissionMonitoring
```

Comment:
```suggestion
            "SendPilotSubmissionMonitoring", self.sendSubmissionMonitoring
```

Reply:
This option should be added also in ConfigTemplate.cfg like https://github.com/DIRACGrid/DIRAC/blob/integration/src/DIRAC/WorkloadManagementSystem/ConfigTemplate.cfg#L278-L279